src/util.rs

```diff
@@ -128,3 +132,41 @@ pub fn output_report(report: &CrashReport, matches: &ArgMatches, argv: &[&str])
     }
     Ok(())
 }
+
+pub fn change_ignored_frames(path: &Path) -> error::Result<()> {
```

Comment:
We need to rename this. Maybe "add_custom_ignored_frames"? Return  just Result<()>.